### PR TITLE
Expose gltfio library so it can be used by third_parties

### PR DESCRIPTION
**Pull request by bejado**
Coming soon: iOS demo

### CODE REVIEW

File: libs/gltfio/src/AssetLoader.cpp [hunk 1]
```diff
@@ -46,6 +46,9 @@
 #define CGLTF_IMPLEMENTATION
 #include <cgltf.h>
 
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb_image.h>
```
Comment (prideout) [inline libs/gltfio/src/AssetLoader.cpp]: `AssetLoader` doesn't use STB, can we fix this another way, perhaps by adding the define to `ResourceLoader` instead?

Comment (bejado) [reply libs/gltfio/src/AssetLoader.cpp]: Ah, yes, that does make more since 👍

**Comment by prideout** [comment]
Approved if we can get kokoro to pass :)

**Comment by bejado** [comment]
Ugh, I was worried about duplicate symbols from STB.. I'll have to rethink this.

**Pull request merged**